interface/GoodParticle.h

```diff
@@ -0,0 +1,28 @@
+#include <TLorentzVector.h>
+
+// Wrapper for lepton objects so to ignore problems with
+// accessing data if particle is muon vs electron
+
+enum class pType {Muon, Electron};
+
+struct GoodPart {
+  TLorentzVector v;
```

Comment:
My preference has been to use ROOT::Math::LorentzVector instead of TLorentzVector, since it's more space and performance optimized: https://root.cern.ch/doc/master/Vector.html

If you want you can add this typedef or include the whole helpers.h file: https://github.com/kdlong/VVAnalysis/blob/master/interface/helpers.h#L8

Reply:
ah! That's how you handle it! I had seen the notes on it in the TLorentzVector class, but it was confusing to see how to implement it. I'll give it a go